src/Configuration/ValueObject/ExportConfig.php

```diff
@@ -116,6 +127,11 @@ public function getTable(): InputTable
         return $this->table;
     }
 
+    public function isIncrementalLoading(): bool
```

Comment:
A prislusny getter, ... po tomto PR to fixnem v `db-extractor-common` a nasledne aktualizuje v `db-extractor-mysql` a ostatnych extractoroch, ktore som uz aktualizoval predtym.